packages/panzer/disc-fe/src/evaluators/Panzer_GatherSolution_BlockedTpetra.hpp

```diff
@@ -147,11 +147,19 @@ class GatherSolution_BlockedTpetra<panzer::Traits::Residual,TRAITS,S,LO,GO,NodeT
   typedef Tpetra::Import<LO,GO,NodeT> ImportType;
   typedef Tpetra::Export<LO,GO,NodeT> ExportType;
 
-  // maps the local (field,element,basis) triplet to a global ID
-  // for scattering
-  Teuchos::RCP<const BlockedDOFManager<LO,GO> > gidIndexer_;
+  //! Maps the local (field,element,basis) triplet to a global ID for
+  // scattering
```

Comment:
Is a ! needed on this second line?

Reply:
that adds the comment to doxygen

Reply:
sorry - not sure. Let me test

Reply:
yeah it needs it. will update after Eric reviews...